storage/innobase/xtrabackup/doc/source/how_xtrabackup_works.rst

```diff
@@ -22,41 +22,49 @@ this, so if the files are changing, then they reflect the state of the database
 at different points in time. At the same time, |Percona XtraBackup| runs a
 background process that watches the transaction log files, and copies changes
 from it. |Percona XtraBackup| needs to do this continually because the
-transaction logs are written in a round-robin fashion, and can be reused after
-a while. |Percona XtraBackup| needs the transaction log records for every
-change to the data files since it began execution.
+transaction logs are written in a round-robin fashion, and can be reused after a
+while. |Percona XtraBackup| needs the transaction log records for every change
+to the data files since it began execution.
 
-|Percona XtraBackup| will use `Backup locks
-<https://www.percona.com/doc/percona-server/5.6/management/backup_locks.html>`_
+|Percona XtraBackup| uses `Backup locks
+<https://www.percona.com/doc/percona-server/8.0/management/backup_locks.html>`_
 where available as a lightweight alternative to ``FLUSH TABLES WITH READ
-LOCK``. This feature is available in |Percona Server| 5.6+. |Percona
-XtraBackup| uses this automatically to copy non-InnoDB data to avoid blocking
-DML queries that modify |InnoDB| tables. When backup locks are supported by the
-server, |xtrabackup| will first copy |InnoDB| data, run the ``LOCK TABLES FOR
-BACKUP`` and copy the |MyISAM| tables and :term:`.frm` files. Once this is
-done, the backup of the files will begin. It will backup :term:`.frm`,
-:term:`.MRG`, :term:`.MYD`, :term:`.MYI`, :term:`.TRG`, :term:`.TRN`,
-:term:`.ARM`, :term:`.ARZ`, :term:`.CSM`, :term:`.CSV`, ``.par``, and
-:term:`.opt` files.
+LOCK``. This feature is available in |Percona Server| 5.6+. |MySQL| 8.0 allows
+acquiring an instance level backup lock via the ``LOCK INSTANCE FOR BACKUP``
```

Comment:
xtrabackup is trying to avoid backup locks and FTWRL when instance contains only InnoDB tables. In this case xtrabackup obtains binary log coordinates from performance_schema.log_status. FTWRL is still needed in MySQL 8 when --slave-info is specified (https://jira.percona.com/browse/PXB-1627). PS 8 extended log_status, so that it also includes relay log coordinates and no locks needed even with --slave-info (https://jira.percona.com/browse/PS-4843)